std/uni.d

```diff
@@ -7132,24 +7136,12 @@ int icmp(S1, S2)(S1 r1, S2 r2)
 }
 
 ///
-@safe unittest
+@safe @nogc nothrow unittest
 {
     assert(icmp("Rußland", "Russland") == 0);
     assert(icmp("ᾩ -> \u1F70\u03B9", "\u1F61\u03B9 -> ᾲ") == 0);
 }
 
-/**
- * By using $(REF byUTF, std,utf) and its aliases, GC allocations via auto-decoding
- * and thrown exceptions can be avoided, making `icmp` `@safe @nogc nothrow pure`.
- */
-@safe @nogc nothrow pure unittest
```

Comment:
It was removed as the unittest is now redundantly identical to the one above it.